modules/OFStateManager/module/src/gentable_handlers.c

```diff
@@ -131,6 +131,7 @@ static indigo_core_op_context_t op_ctx;
 
 /* outstanding op_ctx list for debug purpose */
 static list_head_t outstanding_op_ctx_list;
+static int gentable_handler_init_done = 0;
```

Comment:
this can be a bool

Reply:
It seems lots of module use "ini". I changed it to bool.